### DISCUSSION TITLE
Menu Item Id

### DISCUSSION THREAD

**Discussion by josteph** [chakra-ui/chakra-ui#8066] [Q&A]
Hello, is there a way to reliably get the menu item's id? 
I have a requirement to programmatically click a specific menu item.

<img width="505" alt="image" src="https://github.com/chakra-ui/chakra-ui/assets/13672022/7bd4043b-e4e9-416b-a10f-4a7726194c51">

I can't seem to override it for some reason.

Packages:
- `@chakra-ui/react: 2.7.0`
- `@chakra-ui/menu: 2.1.15`

**Comment by victoriacesar** [comment]
Hello, you found a solution for this case? I want to replace the id and i'm not getting it.

**Reply by josteph** [reply]
No, in the end I had to extend Chakra's `Menu` component
